proton_driver/columns/util.py

```diff
@@ -0,0 +1,61 @@
+
+def get_inner_spec(column_name, spec):
+    brackets = 0
+    offset = len(column_name)
+
+    for i, ch in enumerate(spec[offset:], offset):
+        if ch == '(':
+            brackets += 1
+
+        elif ch == ')':
+            brackets -= 1
+
+        if brackets == 0:
+            break
+
+    return spec[offset + 1:i]
+
+
+def get_inner_columns(spec):
```

Comment:
the name 'inner' is a little confusing. Looks it getting first level columns (no inner concept here)

consider to rename and/or add comments to explain these util functions.